### ISSUE TITLE
START on login but startup as DISABLED

### ISSUE THREAD

**Issue by mwoffenden** [ianyh/Amethyst#828]
## System

 - **Amethyst version**: 0.14.0b
 - **macOS version**: 10.14.4

## What's the problem?
Will there be an option to START on login but startup as DISABLED?

**Comment by mwoffenden** [comment]
Will this be fixed anytime soon?  

I'm sure many, myself includes, would love to see auto start in Diabled mode working.

**Comment by ianyh** [comment]
I mislabeled this, but it’s not really top of the list.

**Comment by orlandoaleman** [comment]
I use keyboard shortcuts to move windows between different desktops very often, but auto tiling not so often. It would be very useful for me to be able to start the app with this disabled.

Greetings and thanks for the work you do.

**Comment by bitbuyandrew** [comment]
I'd love to see Amethyst start with disabled mode. I use it frequently enough but most times I need my mac os workspaces to remain as is. to have control over when/where my browser windows get re-arranged is paramount for me.
Thank you

**Comment by vietstone-ng** [comment]
Love to see this too

**Comment by sagemintblue** [comment]
I'd also like to have this feature. Today I read through the configuration files documentation [1] looking for some way to accomplish this.

[1] https://github.com/ianyh/Amethyst/blob/development/docs/configuration-files.md

**Comment by ianyh** [comment]
Yeah, it's worth doing, and I would accept a change from someone, but my focus has mostly been on bugs and custom layout support.
